packages/matter-node.js/src/tsconfig.json

```diff
@@ -35,6 +35,10 @@
             "util/export.ts",
             "export.ts"
         ],
+        "plugin": ["typedoc-plugin-missing-exports", "typedoc-plugin-markdown"],
+        "excludeExternals": true,
+        "parametersFormat": "table",
+        "membersWithOwnFile": ["Class", "Enum", "Interface"],
```

Comment:
iexperiment aroud, maybe we can use better the compressed in one file thing for now